react/components/Title/index.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+
+import { LEFT, RIGHT, CENTER } from './constants.js'
+import title from './title.css'
+
+/**
+ * Name component. Show name and relevant SKU information of the Product Summary
```

Comment:
This description does not look quite right 🤔

Reply:
Done. :sunglasses: